xbmc/settings/windows/GUIWindowSettingsScreenCalibration.cpp

```diff
@@ -245,9 +283,38 @@ void CGUIWindowSettingsScreenCalibration::EnableControl(int iControl)
   SET_CONTROL_VISIBLE(CONTROL_BOTTOM_RIGHT);
   SET_CONTROL_VISIBLE(CONTROL_SUBTITLES);
   SET_CONTROL_VISIBLE(CONTROL_PIXEL_RATIO);
+  SET_CONTROL_VISIBLE(CONTROL_RESET);
   SET_CONTROL_FOCUS(iControl, 0);
 }
 
+void CGUIWindowSettingsScreenCalibration::StoreControlsSize()
```

Comment:
can you explain the need for storing control sizes? I mean the fact this is needed sounds suspicious/hacky. Maybe what is needed is the persistence of some other values those controls refer to, not really the controls themselves

Reply:
Ok, I see this doesn't really store anything anywhere, just memory. I think this can be private as it is specific to this window implementation. Please also check if you don't need some sort of locking on the map (probably the increased read consistency - you only write the values here - won't justify the impact on performance and GUI locks are bad anyway.